test/e2e/utils/pods/rtepod/rtepod.go

```diff
@@ -65,21 +65,14 @@ func FindNotificationFilePath(rtePod *corev1.Pod) (string, error) {
 	for idx := 0; idx < len(rtePod.Spec.Containers); idx++ {
 		cnt := rtePod.Spec.Containers[idx] // shortcut
 		if len(cnt.Command) > 0 && strings.Contains(cnt.Command[0], rteExecutable) {
-			for _, arg := range cnt.Command {
-				if strings.Contains(arg, notificationFileOption) {
-					return extractOptionValue(arg)
-				}
-			}
-		}
-		if len(cnt.Args) > 0 && strings.Contains(cnt.Args[0], rteExecutable) {
 			for _, arg := range cnt.Args {
 				if strings.Contains(arg, notificationFileOption) {
 					return extractOptionValue(arg)
 				}
 			}
 		}
 	}
-	return "", fmt.Errorf("no container uses %q as command or argument option", notificationFileOption)
+	return "", fmt.Errorf("no container uses %q as an argument option", notificationFileOption)
```

Comment:
this is a worthy improvement, but I don't think will fix the flakiness, at least not completely.

Reply:
Why not, the notification file test is always falling at this point. I checked the logs

Reply:
The whole patch should fix it, not this line specifically